salt/metalk8s/metalk8s-ui/deployed.sls

```diff
@@ -1,5 +1,20 @@
 {% set kubeconfig = "/etc/kubernetes/admin.conf" %}
 {% set context = "kubernetes-admin@kubernetes" %}
+{%- set control_plane_ips = salt.saltutil.runner('mine.get', tgt='*', fun='control_plane_ips') %}
+{%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
+{%- set control_plane_ip = control_plane_ips[pillar['bootstrap_id']][0] %}
+{%- else %}
+{%- set control_plane_ip = 'localhost' %}
+{%- endif %}
+
+Create metalk8s-ui directories:
+  file.directory:
```

Comment:
When is this directory used?

Reply:
to create ``ui```in ```metalk8s``` directory

Reply:
Sure, but why do you need that directory to exist (on the host)?